README.md

```
@@ -2,8 +2,8 @@
 
 ![EduBridge Logo](https://github.com/user-attachments/assets/c811683b-575e-44d4-a31c-d9661b6c3c60)
 
-  
-# 🎓 EduBridge  
+
+# 🎓 EduBridge
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)